### DISCUSSION TITLE
WHAT am I doing wrong?

### DISCUSSION THREAD

**Discussion by 747skipper** [mifi/lossless-cut#2104] [Support]
First; thanks for some GREAT software.  Being able to edit video with its subtitles is so rare and just what I want to do,  But I'm falling at a simple hurdle.
How do I make the picture move with the playhead?
I can move the play head with the mouse using point and click, drag or scroll wheel, by clicking one frame forward or back or one key frame forward or back but I can't move the picture with it.
Surly there must be a way to move the picture?

**Comment by mifi** [comment]
hi. so the picture doesn't move at all? does this happen with all kinds of video files? which operating system are you on?

**Reply by 747skipper** [reply]
Thanks Mifi.  Sorry for delay holidays here.
I'm on Windows 10 and it's an AVI file.
BUT
I've just restarted and it's working fine .
If it happens again I'll let you know and try to add more details
All the best
David
